src/components/cvaa-overlay/custom-captions-window.tsx

```diff
@@ -1,204 +1,207 @@
-            labelId="cvaa.background_color_label"
-            options={backgroundColorOptions}
-            classNames={[style.formGroupRow, style.backgroundColor]}
-            styleName="backgroundColor"
-            changeCustomStyle={props.changeCustomStyle}
-          />
-          <SliderCaptionsStyle
-            addAccessibleChild={props.addAccessibleChild}
-            labelId="cvaa.background_opacity_label"
-            value={props.customTextStyle.backgroundOpacity}
-            classNames={[style.formGroupRow, style.backgroundOpacity]}
-            styleName="backgroundOpacity"
-            changeCustomStyle={props.changeCustomStyle}
-          />
-          <div className={style.formGroupRow}>
-            <a
-              role="button"
-              tabIndex={0}
-              ref={el => {
-                props.addAccessibleChild(el);
-              }}
-              onClick={() => {
-                this.changeCaptionsStyle();
-                this.transitionToState();
-              }}
-              onKeyDown={this.onKeyDown}
-              className={[style.btn, style.btnBranded, style.btnBlock].join(' ')}
-            >
-              <Text id={'cvaa.apply'} />
-            </a>
-          </div>
-
-          <div className={style.previewContainer}>
-            <span style={props.getPreviewStyle()}>
-              <Text id={'cvaa.caption_preview'} />
-            </span>
-          </div>
-        </form>
-      </div>
-    );
-  }
-}
-
-export {CustomCaptionsWindow};
+import {Component, VNode} from 'preact';
+import style from '../../styles/style.scss';
+import {KeyMap} from '../../utils/key-map';
+import {Text} from 'preact-i18n';
+import {h} from 'preact';
+import {DropDownCaptionsStyle} from './drop-down-captions-style';
+import {SliderCaptionsStyle} from './slider-captions-style';
+import {withPlayer} from '../player';
+
+/**
+ * CustomCaptionsWindow component
+ * @class CustomCaptionsWindow
+ * @extends {Component}
+ */
+@withPlayer
+class CustomCaptionsWindow extends Component<any, any> {
+  /**
+   * after component mounted, set focus on default
+   *
+   * @returns {void}
+   * @memberof CustomCaptionsWindow
+   */
+  componentDidMount(): void {
+    this.props.focusOnDefault();
+  }
+
+  /**
+   * change captions style handler
+   *
+   * @returns {void}
+   * @memberof CustomCaptionsWindow
+   */
+  changeCaptionsStyle = (): void => {
+    this.props.changeCaptionsStyle(this.props.customTextStyle);
+  };
+
+  /**
+   * transition to state handler
+   *
+   * @returns {void}
+   * @memberof MainWindow
+   */
+  transitionToState = (): void => {
+    this.props.transitionToState(this.props.cvaaOverlayState.Main);
+  };
+
+  /**
+   * on key down handler
+   *
+   * @param {KeyboardEvent} e - keyboard event
+   * @returns {void}
+   * @memberof CustomCaptionsWindow
+   */
+  onKeyDown = (e: KeyboardEvent): void => {
+    if (e.keyCode === KeyMap.ENTER || e.keyCode === KeyMap.SPACE) {

```

Comment:
Please re apply this fix and then save with "change without formatting"
Otherwise it will be very hard in the future to understand what the change was when looking at the commit because the entire file appears modified